app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -99,8 +112,32 @@ fun RunHikeScreen(
 }
 
 @Composable
-private fun RunHikeContent(hike: DetailedHike, navigationActions: NavigationActions) {
+private fun RunHikeContent(
+    hike: DetailedHike,
+    navigationActions: NavigationActions,
+    facilitiesViewModel: FacilitiesViewModel
+) {
   // Avoids the app crashing when spamming the back button
+
+  val facilities = remember { mutableStateOf<List<Facility>?>(null) }
```

Comment:
I moved the facilities list to the `viewModel`.
The reason I didn't do it before:
I didn't want to adapt the whole `viewModel` to my implementation of the Screens, since this `viewModel` could be used elsewhere. I adapted it by making it still possible to be used as before just with a function more specific to hikes.